tools/nuget/nuget.proj

```diff
@@ -29,6 +29,10 @@
       <None Include="..\..\$(Platform)\$(Configuration)\ntosebpfext.pdb" PackagePath="build\native\bin" Pack="true" />
       <None Include="..\..\$(Platform)\$(Configuration)\neteventebpfext.sys" PackagePath="build\native\bin" Pack="true" />
       <None Include="..\..\$(Platform)\$(Configuration)\neteventebpfext.pdb" PackagePath="build\native\bin" Pack="true" />
+      <None Include="..\..\include\ebpf_netevent_hooks.h" PackagePath="build\native\include" Pack="true" />
```

Comment:
Do you also want to add the ntosext header and export_program_info.exe/pdb?

Reply:
had added script only for neteventebpfext.
added script for ntosebpfext as well now.